UI/EndTurnButton.gd

```diff
@@ -5,10 +5,12 @@ extends Button
 func _process(delta: float) -> void:
```

Comment:
delta is not used, should add a `@warning_ignore("unused_parameter")`
I saw this problem in multiple other files, so i'll probably open an issue for that

Reply:
Closing this, will do everything in a dedicated issue